scripts/ci/tags.yaml

```diff
@@ -107,261 +105,288 @@ ci_samples_net: &ci_net
     - nrf/subsys/dfu/
     - nrf/subsys/fw_info/
     - nrf/subsys/net/
     - nrf/subsys/nrf_security/
     - nrf/subsys/partition_manager/
-    - bootloader/mcuboot/
-    - modules/crypto/mbedtls/
-    - modules/crypto/oberon-psa-crypto/
-    - modules/lib/azure-sdk-for-c/
-    - modules/lib/cjson/
-    - modules/lib/hostap/
-    - modules/lib/zcbor/
+    - nrf/sysbuild/
+    - nrf/tests/subsys/net/
     - nrfxlib/crypto/
     - nrfxlib/nrf_modem/
     - nrfxlib/nrf_wifi/
+    - zephyr/cmake/
+    - zephyr/drivers/ethernet/
+    - zephyr/drivers/ieee802154/
+    - zephyr/drivers/net/
+    - zephyr/drivers/ptp_clock/
+    - zephyr/drivers/wifi/
+    - zephyr/include/zephyr/net/
+    - zephyr/share/
+    - zephyr/subsys/mgmt/
+    - zephyr/subsys/net/
+    - zephyr/subsys/random/
+    - zephyr/subsys/settings/
+    - zephyr/subsys/shell/
+    - zephyr/subsys/storage/
 
 ci_tests_subsys_net: *ci_net
 
 ci_samples_wifi:
   files:
+    - modules/lib/hostap/
     - nrf/drivers/wifi/
     - nrf/modules/hostap/
     - nrf/samples/wifi/
-    - modules/lib/hostap/
-    - zephyr/subsys/net/l2/wifi/
-    - zephyr/subsys/net/l2/ethernet/
-    - nrfxlib/nrf_wifi/
     - nrfxlib/crypto/
+    - nrfxlib/nrf_wifi/
+    - zephyr/subsys/net/l2/ethernet/
+    - zephyr/subsys/net/l2/wifi/
 
 ci_applications_matter: &ci_matter
   files:
-    - nrf/drivers/mpsl/
+    - modules/lib/hostap/
+    - modules/lib/matter/
+    - modules/lib/suit-generator/
+    - nrf/applications/matter_bridge/
+    - nrf/applications/matter_weather_station/
+    - nrf/cmake/
     - nrf/drivers/hw_cc3xx/
+    - nrf/drivers/mpsl/
     - nrf/drivers/wifi/
     - nrf/modules/hostap/
-    - nrf/subsys/mpsl/
-    - nrf/subsys/ieee802154/
+    - nrf/modules/mcuboot/
+    - nrf/modules/nrfxlib/nrf_802154/
+    - nrf/samples/matter/
+    - nrf/samples/nrf5340/multiprotocol_rpmsg/
+    - nrf/snippets/matter-diagnostic-logs/
+    - nrf/subsys/bluetooth/controller/
     - nrf/subsys/bootloader/
-    - nrf/subsys/partition_manager/
     - nrf/subsys/dfu/
     - nrf/subsys/fw_info/
-    - nrf/subsys/pcd/
-    - nrf/subsys/bluetooth/controller/
+    - nrf/subsys/ieee802154/
+    - nrf/subsys/mpsl/
     - nrf/subsys/net/openthread/
     - nrf/subsys/nrf_security/
-    - nrf/modules/nrfxlib/nrf_802154/
-    - nrf/modules/mcuboot/
-    - nrf/snippets/matter-diagnostic-logs/
-    - nrf/samples/matter/
-    - nrf/applications/matter_weather_station/
-    - nrf/applications/matter_bridge/
-    - nrf/samples/nrf5340/multiprotocol_rpmsg/
-    - nrfxlib/openthread/
+    - nrf/subsys/partition_manager/
+    - nrf/subsys/pcd/
+    - nrf/sysbuild/
+    - nrfxlib/crypto/
     - nrfxlib/nrf_802154/driver/
     - nrfxlib/nrf_802154/serialization/
     - nrfxlib/nrf_802154/sl/
     - nrfxlib/nrf_wifi/
-    - nrfxlib/crypto/
+    - nrfxlib/openthread/
+    - zephyr/cmake/
+    - zephyr/drivers/net/
     - zephyr/include/dfu/
     - zephyr/include/mgmt/mcumgr/
+    - zephyr/samples/bluetooth/hci_ipc/
```

Comment:
@kkasperczyk-no, do you use HCI IPC sample for Matter? NCS already uses IPC radio application by default, so this one seems outdated

Reply:
I believe they still use the hci_ipc for some wifi coex sample variant.

Reply:
Will keep it.